lib/ui/screens/home_page.dart

```diff
@@ -66,81 +74,157 @@ class HomePageState extends ConsumerState<HomePage> {
+          halfWidth + internalRadius * sin(step + halfDegreesPerStep),
+        );
+    }
+    path.close();
+    return path;
+  }
+
   @override
   Widget build(BuildContext context) {
     List<Task> items = ref.watch(taskStateProvider).valueOrNull ?? [];
-    return Scaffold(
-      body: CustomScrollView(
-        controller: scrollController,
-        physics: const BouncingScrollPhysics(),
-        slivers: [
-          SliverPersistentHeader(
-            floating: true,
-            pinned: true,
-            delegate: _AppHeader(
-              completedTaskCount:
-                  ref.read(taskStateProvider.notifier).countDoneTasks(),
-              isConnected: _connectionStatus[0] == ConnectivityResult.none
-                  ? true
-                  : false,
-              isVisible: isVisible ? true : false,
-              onChangeVisibility: () {
-                setState(() {
-                  isVisible = !isVisible;
-                });
-              },
-            ),
-          ),
-          SliverToBoxAdapter(
-            child: Card(
-              color: Theme.of(context).cardColor,
-              elevation: 2,
-              margin: const EdgeInsets.only(right: 16, left: 16),
-              child: Column(
-                children: [
-                  const SizedBox(
-                    height: 8,
-                  ),
-                  ListView.builder(
-                    itemCount: items.length,
-                    controller: scrollController,
-                    shrinkWrap: true,
-                    itemBuilder: (BuildContext context, int index) {
-                      if ((isVisible && items[index].isDone)) {
-                        return const SizedBox();
-                      }
-                      return TaskCard(
-                        task: items[index],
-                        isVisible: isVisible,
-                      );
-                    },
-                  ),
-                  SingleChildScrollView(
-                    child: NewTaskCard(),
-                  ),
-                  const SizedBox(
-                    height: 8,
+    return Stack(
+      children: [
+        Scaffold(
+          body: CustomScrollView(
+            controller: scrollController,
+            physics: const BouncingScrollPhysics(),
+            slivers: [
+              SliverPersistentHeader(
+                floating: true,
+                pinned: true,
+                delegate: _AppHeader(
+                  completedTaskCount:
+                      ref.read(taskStateProvider.notifier).countDoneTasks(),
+                  isConnected: _connectionStatus[0] == ConnectivityResult.none
+                      ? true
+                      : false,
+                  isVisible: isVisible ? true : false,
+                  onChangeVisibility: () {
+                    setState(() {
+                      isVisible = !isVisible;
+                    });
```

Comment:
можно сделать одной строкой
```suggestion
                    setState(() => isVisible = !isVisible);
```